public/app/percona/dbaas/components/DBCluster/AddDBClusterModal/DBClusterBasicOptions/DBClusterBasicOptions.tsx

```diff
@@ -56,6 +62,31 @@ export const DBClusterBasicOptions: FC<DBClusterBasicOptionsProps> = ({ kubernet
     change(AddDBClusterFields.kubernetesCluster, selectedKubernetes);
   }, []);
 
+  const isDatabaseVersionDisabled = useMemo(() => isOptionEmpty(databaseType), [databaseType]);
+
+  const getDatabaseVersions = async () => {
```

Comment:
Probably for simplicity purposes it could be moved to custom hook?

Reply:
Done!